src/Metrics.jl

```diff
@@ -193,10 +194,10 @@ function update!(meanflow::MeanFlow, flow::Flow)
 end
 
 uu!(τ,a::MeanFlow) = for i in 1:ndims(a.P), j in 1:ndims(a.P)
-    @loop τ[I,i,j] = a.UU[I,i,j] - a.U[I,i,j] * a.U[I,i,j] over I in CartesianIndices(a.P)
+    @loop τ[I,i,j] = a.UU[I,i,j] - a.U[I,i] * a.U[I,j] over I in CartesianIndices(a.P)
```

Comment:
Should we also check that `a.UU` exists before doing this? Since users can initialise the `MeanFlow` without `UU`?

Reply:
This function is just a helper for users, not really part of the computation of the mean flow statistics. So if they want to use it, they should have also stored `UU` before. An error will pop up otherwise, which is fine